addons/mail/static/src/core/common/message.xml

```diff
@@ -55,7 +55,7 @@
                             </small>
                             <div t-if="props.message.scheduledDate" t-att-class="{ 'ms-2': props.isInChatWindow and isAlignedRight }" t-att-title="messageService.scheduledDateSimple(props.message)">
                                 <span class="text-600 cursor-pointer">
-                                    <i class="fa fa-calendar-o"/>
+                                    <i class="fa fa-calendar-o ms-1"/>
```

Comment:
According to the this [line](https://github.com/odoo/odoo/pull/151700/files#diff-0fc14194d473213159537bf41f421dcbaf940dfd81ac87d263b196e914d40af3R56), extra space will be added to the chat window.